src/lang/en.json

```diff
@@ -188,6 +188,14 @@
   "FIELD_FLAG_6": "Flag 6",
   "FIELD_FLAG_7": "Flag 7",
   "FIELD_FLAG_8": "Flag 8",
+  "FIELD_FLAG_8": "Flag 9",
```

Comment:
those should be FIELD_FLAG_9 ... 16

Reply:
Right, I forgot to hit ctrl-s after copy/pasting. 😄